src/roompanel/RoomManager.cpp

```diff
@@ -0,0 +1,252 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (C) 2021 The MMapper Authors
+// Author: Massimiliano Ghilardi <massimiliano.ghilardi@gmail.com> (Cosmos)
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <QtCore>
+
+#include "../global/StringView.h"
+#include "../proxy/GmcpMessage.h"
+#include "RoomManager.h"
+#include "RoomMobs.h"
+
+RoomManager::RoomManager(QObject *const /*parent*/)
+    : QObject{nullptr}
```

Comment:
Why not pass in the QMainWindow as the parent? This is will otherwise be leaked when the application is closed.

Reply:
I'm not an expert of Qt objects lifecycle... I will fix this.

Reply:
Fixed